mir-ci/mir_ci/display_server.py

```diff
@@ -60,6 +63,13 @@ def program(self, app: App, env: Dict[str, str] = {}) -> Program:
             app, env=dict({"DISPLAY": "no", "QT_QPA_PLATFORM": "wayland", "WAYLAND_DISPLAY": self.display_name}, **env)
         )
 
+    def environment(self, key: str, value: str):
+        """
+        Set an environment variable
+        """
+        self.env[key] = value
+        return self
+
```

Comment:
I would rather we add `env: Dict[str, str]` to `__init__()`, just like we have with `Program`:

https://github.com/MirServer/mir-ci/blob/a453ae04cfc644b62d785de22405596e6505736c/mir-ci/mir_ci/program.py#L43